requirements.txt

```diff
@@ -9,7 +9,7 @@ autograd~=1.4
 toml~=0.10
 appdirs~=1.4
 semantic_version~=2.10
-autoray>=0.6.1,<0.6.10
+autoray>=0.6.1,!=0.6.10
```

Comment:
Do we want to do this? Are we actually supporting autoray versions 0.6.1-0.6.8, still?

Reply:
Give me a few minutes and I'll run the all-interfaces tests. Anything else?

Reply:
Tests resulted in the insight that the PL test suite with latest dependencies (like TF) installed will only succeed with Autoray 0.6.11. Therefore I'll pin it to `>=0.6.11`.
@albi3ro for visibility, although you approved already.
@trbromley Is this fine by you?

Reply:
Thanks @dwierichs! I believe `requirements.txt` is intended to be pinned fairly specifically, while `setup.py` should be as loose as possible. So here we could be more specific than `>=0.6.11` (or 12).

Reply:
I changed both to `>=0.6.11` because of the failures found by @vincentmr . I don't think the `setup.py` should include versions that don't work by default (i.e. with default TF version)

Reply:
Agreed! I meant more that we could go with an `==` pin. Though looking at the other pins in this file we go with `~=` so I'm not sure. @albi3ro or @mlxd might have a better picture.